sycl/test/basic_tests/buffer/buffer_dev_to_dev.cpp

```diff
@@ -4,6 +4,9 @@
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
 
+// Unsupported: cuda
+//  The test fails sporadically on cuda. See https://github.com/intel/llvm/issues/1508 for more details.
+
```

Comment:
Please, revert changes in this file.
https://github.com/intel/llvm/commit/d9b2d526420d168a626b454e5592d04e92da629a already disabled the test.

Reply:
Do I need to do a rebase?

Reply:
No. CI system does the merge with tip of the sycl branch before testing.